bundle/NOTICE.txt

```diff
@@ -1,5 +1,5 @@
 Copyright (c) "Neo4j"
-Neo4j Sweden AB [http://neo4j.com]
+Neo4j Sweden AB [https://neo4j.com]
 
 This file is part of Neo4j.
```

Comment:
You might forgot to remove this phrase from this header.

Reply:
Same as here: https://github.com/neo4j/neo4j-java-driver/pull/1496#discussion_r1371809592